packages/core/package.json

```diff
@@ -118,5 +118,9 @@
   },
   "msw": {
     "workerDirectory": "public"
+  },
+  "volta": {
```

Comment:
Why on others you extended the volta configuration from the monorepo but not here?

Reply:
Because this code is shipped to the `.faststore` folder. I tried putting it there as well, but when we run `yarn dev` or `yarn build` there, `volta` tries to find the extended configuration and it fails, because the monorepo package.json is not there.